website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,192 @@
+
+**Filled**
+
+When one or more values are selected, the trigger is populated with the selected value(s). One or more values can be pre-selected before user interaction, as needed.
+
+![Showing preselected options in both a single and multiple variant. This helps show something has been preselected previously and the user can update their options](/assets/components/form/super-select/super-select-trigger-filled.png)
+
+## Before and after options
+
+The before and after options are areas above and below the list, independent from the list of options. Predefined components (like search) or custom content can occupy these spaces.
+
+### BeforeOptions
+
+BeforeOptions allows for either a search field or custom content to be displayed before the options list. Applicable to both Single and Multiple components.
+
+!!! Warning
+
+Currently, the Multiple component only supports search in the trigger and is considered an accessibility issue (nested interactive). This is something we are working to improve in collaboration with the PowerSelect team.
+
+!!!
+
+#### Search
+
+If enabled, search allows the user to filter results based on a query.
+
+![Search enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-before-options-search.png =836x*)
+
+#### CustomContent
+
+If enabled, custom content will occupy this space.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-before-options-custom-content.png =836x*)
+
+!!! Dont
+
+We recommend against placing heavy UI elements in custom content as it can draw attention away from the list options and will only be visible when the menu is open. Consider placing them in outside the list relative to the trigger for better visibility.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-dont-visually-heavy.png =836x*)
+
+!!!
+
+### AfterOptions
+
+AfterOptions allows for either an HDS standardized footer or custom content to be displayed after the options list. Applicable to both Single and Multiple variants, although the footer has additional functionality in the Multiple variant.
+
+#### Footer
+
+By default, the footer displays below the list and contains content and/or buttons (depending on the variant) that provide additional context (or functions) to the list options.
+
+##### Single
+
+The footer shows a total count, which is customizable to fit the context.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-single-variant-footer-with-counter.png)
+
+##### Multiple
+
+In the multiple select variant, the `afterOptions` and `footer` display buttons to expedite the deselection process and optionally a counter (selected / total).
```

Comment:
RE: "expedite the deselection process" This sounds really klunky to me... 

I would explain more simply what the buttons do.

Reply:
Left this paragraph as a summary because the next paragraphs explain what those buttons do.

Reply:
How about "modify selections" at least vs. "expedite the deselection process"? @jorytindall any opinions?

Reply:
This is a tricky one, maybe something like:

> In the multiple select variant, the `afterOptions` and `footer` display buttons to modify the list of options and optionally a counter (selected / total).

WDYT?

Reply:
Going to make this:

"In the Multiple select component, the `afterOptions` and `footer` display buttons modify the list of options and display a counter (selected / total)."

(Feel free to suggest further edits.)

Reply:
Ok I think I have something better now. I simplified it to follow the description for the footer in the Single component:

"The footer displays buttons to modify the list of options and a counter of the selected total options."